### ISSUE TITLE
Cred Manager Script is failing to initialize in container

### ISSUE THREAD

**Issue by afroDC** [GluuFederation/docker-oxauth#7]
The following error is from `Manage Custom Scripts` -> `Cred Manager` -> `Show Error`

```
org.xdi.exception.PythonException: Failed to load python file 'java.io.ByteArrayInputStream@33fcffda'
	at org.xdi.service.PythonService.loadPythonScript(PythonService.java:182)
	at org.xdi.service.PythonService$Proxy$_$$_WeldClientProxy.loadPythonScript(Unknown Source)
	at org.xdi.service.custom.script.CustomScriptManager.createExternalTypeFromStringWithPythonException(CustomScriptManager.java:372)
	at org.xdi.service.custom.script.CustomScriptManager$Proxy$_$$_WeldSubclass.createExternalTypeFromStringWithPythonException(Unknown Source)
	at org.xdi.service.custom.script.CustomScriptManager.createExternalType(CustomScriptManager.java:340)
	at org.xdi.service.custom.script.CustomScriptManager.reloadCustomScriptConfigurations(CustomScriptManager.java:253)
	at org.xdi.service.custom.script.CustomScriptManager.reloadImpl(CustomScriptManager.java:153)
	at org.xdi.service.custom.script.CustomScriptManager.reload(CustomScriptManager.java:141)
	at org.xdi.service.custom.script.CustomScriptManager.initTimer(CustomScriptManager.java:103)
	at org.xdi.service.custom.script.CustomScriptManager$Proxy$_$$_WeldSubclass.initTimer$$super(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.jboss.weld.interceptor.proxy.TerminalAroundInvokeInvocationContext.proceedInternal(TerminalAroundInvokeInvocationContext.java:51)
	at org.jboss.weld.interceptor.proxy.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:78)
	at org.xdi.service.cdi.async.AsynchronousInterceptor$1.get(AsynchronousInterceptor.java:36)
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1590)
	at java.lang.Thread.run(Thread.java:748)
Caused by: Traceback (most recent call last):
  File "credmanager.py", line 22, in <module>
ImportError: No module named twilio

	at org.python.core.Py.ImportError(Py.java:334)
	at org.python.core.imp.import_logic(imp.java:913)
	at org.python.core.imp.import_module_level(imp.java:970)
	at org.python.core.imp.importName(imp.java:1057)
	at org.python.core.ImportFunction.__call__(__builtin__.java:1280)
	at org.python.core.PyObject.__call__(PyObject.java:450)
	at org.python.core.__builtin__.__import__(__builtin__.java:1232)
	at org.python.core.imp.importFromAs(imp.java:1149)
	at org.python.core.imp.importFrom(imp.java:1124)
	at org.python.pycode._pyx3.f$0(credmanager.py:61)
	at org.python.pycode._pyx3.call_function(credmanager.py)
	at org.python.core.PyTableCode.call(PyTableCode.java:171)
	at org.python.core.PyCode.call(PyCode.java:18)
	at org.python.core.Py.runCode(Py.java:1614)
	at org.python.util.PythonInterpreter.execfile(PythonInterpreter.java:296)
	at org.xdi.service.PythonService.loadPythonScript(PythonService.java:179)
	... 18 more
```

The import bit seems to be `ImportError: No module named twilio`.

**Comment by afroDC** [comment]
Issue can be fixed in the Dockerfile by making the `/opt/gluu/jetty/oxauth/custom/libs/` directory [according to the documentation](https://gluu.org/docs/ce/authn-guide/sms-otp/#add-twilio-library-to-oxauth) and adding the jar from [this link](http://search.maven.org/remotecontent?filepath=com/twilio/sdk/twilio/7.17.6/twilio-7.17.6.jar).

**Comment by afroDC** [comment]
I propose we make the fix in the Dockerfile and push the `3.1.2_dev` and `3.1.3_dev` images to the repo. Thoughts @iromli ?
